docker/docker-compose.yml

```diff
@@ -0,0 +1,91 @@
+version: '3'
+volumes:
+  girder: {}
+  db: {}
+  worker: {}
+services:
+  docker:
+    image: 'docker:dind'
```

Comment:
So, my interpretation of the same sources leads me to believe that it is not docker-in-docker, itself, that is obsolete; but the old instructions on how to achieve such a setup. ("This work" referring to the repo, not the official "docker" docker image).

"If you want to run Docker-in-Docker today, all you need to do is:"

implies to me that docker-in-docker is still very much supported, but "this is how you do it, now..."

I admit the language and context is vague.